app/controllers/items_controller.rb

```diff
@@ -266,8 +266,9 @@ def rights
       cocina = NilModel.new(params[:id])
     end
 
-    apo_object = Dor.find(@apo)
-    @form = AccessForm.new(cocina, apo_object)
+    form_type = cocina.collection? ? CollectionRightsForm : DRORightsForm
```

Comment:
These next couple lines look to be uncovered.

Reply:
They were uncovered before too.